src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -51,26 +57,48 @@ const AssetList: FunctionComponent<AssetListProps> = ({
   className,
   featureTypes,
 }) => {
-  const { id, type, isReported } = selection
+  const { id, type, isReported, isChecked } = selection
   const { description, icon }: Partial<FeatureType> =
     featureTypes?.find(({ typeValue }) => typeValue === type) ?? {}
 
   const label = [description, id].filter(Boolean).join(' - ')
 
   if (!id) return null
 
+  const reportedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'reported'
+  ) as ReportedFeatureType
+
+  const checkedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'checked'
+  ) as CheckedFeatureType
+
   return (
     <IconList data-testid="assetList" className={className}>
       <IconListItem
         key={id}
-        id={isReported ? `assetListItem-${id}-reported` : `assetListItem-${id}`}
+        id={
+          isReported
+            ? `assetListItem-${id}-reported`
+            : isChecked
+            ? `assetListItem-${id}-checked`
+            : `assetListItem-${id}`
+        }
```

Comment:
Nested ternaries should be avoided; hard to read and difficult to debug

Reply:
In general I agree, but I'm not so fond of the alternative either

Reply:
`  let extendedId = `assetListItem-${id}`
  if (isReported) {
    extendedId = `assetListItem-${id}-reported`
  }
  else if (isChecked) {
    extendedId = `assetListItem-${id}-checked`
  }`